packages/angular/cli/README.md

```diff
@@ -234,26 +234,33 @@ For more informations about Node.js debugging in VS Code, see the related [VS Co
 
 In order to investigate performance issues, CPU profiling is often useful.
 
-To capture a CPU profiling, you can:
+#### Creating a profile
 
-1. install the v8-profiler-node8 dependency: `npm install v8-profiler-node8 --no-save`
-1. set the NG_CLI_PROFILING Environment variable to the file name you want:
-   - on Unix systems (Linux & Mac OS X): ̀`export NG_CLI_PROFILING=my-profile`
-   - on Windows: ̀̀`setx NG_CLI_PROFILING my-profile`
+To capture a CPU profiling, you run the build with `NG_BUILD_PROFILING` environment variable set to `1` or `true`
 
-Then, just run the ng command on which you want to capture a CPU profile.
-You will then obtain a `my-profile.cpuprofile` file in the folder from which you ran the ng command.
+- on Unix systems (Linux & Mac OS X): ̀`NG_BUILD_PROFILING=1 ng build`
+- on Windows: ̀̀`set NG_BUILD_PROFILING=1 && ng build`
+
+You will then obtain a `chrome-profiler-events.json` file in the folder from which you ran the `ng` command.
+
+Or Node.js 16+ users can use the Node.js command line argument `--cpu-prof` to create a CPU profile.
```

Comment:
Is this equivalent to `NG_BUILD_PROFILING=1` or is this a different kind of profile?

Reply:
Same kind